src/util/basketUtils.js

```diff
@@ -75,15 +76,36 @@ export function setLendAmount({ amount, apollo, loanId }) {
 	});
 }
 
-export function setDonationAmount() {
-	// TODO
+export function setDonationAmount({ apollo, donationAmount }) {
```

Comment:
`donationAmount` was formerly `amount` so the actual update was using `0` :(. Fixed!